resources/js/features/authentication/pages/Login.page.tsx

```diff
@@ -0,0 +1,101 @@
+import { SubmitHandler, createForm, zodForm } from "@modular-forms/solid";
+import { Navigate } from "@solidjs/router";
+import { createMutation, useQueryClient } from "@tanstack/solid-query";
+import { Button } from "components/ui";
+import { Page, QueryBarrier } from "components/utils";
+import { User } from "data-access/memo-api";
+import { Component } from "solid-js";
+import { z } from "zod";
+
+namespace LoginForm {
+  export const schema = z.object({
+    email: z
+      .string()
+      .min(1, "Pole jest wymagane")
+      .email("Wprowadź poprawny email"),
+    password: z.string().min(1, "Pole wymagane"),
```

Comment:
Może jakoś te komunikaty błędów walidacji wyciągnąć w jedno miejsce, nawet dla całej aplikacji? Bo na razie brak konsekwencji jest dość mocny, tu jest inaczej niż dwie linie wyżej :)

Reply:
jest do ustalenia i zrobienia mechanizm błędów, ja dalej uważam, że powinny być 2 rodzaje walidacji, frontowa i bacendowa, ale Mikołaj się upiera przy backendowej. No i jeszcze u nas nie ma być prostych przypadków typu opcjonalne wymagane, tylko coś jeszcze z tego co pamiętam. Ujednolicenie formularzy to kolejne zadanie